spec/services/cdl/eligible_item_service_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe Cdl::EligibleItemService do
+  describe ".item_ids" do
+    context "on_cdl is null" do
+      before do
+        stub_request(:get, "https://bibdata.princeton.edu/#{bib_id}/items")
+          .to_return(status: 200,
+                     body: file_fixture("bibdata/#{bib_id}.json").read, headers: { "Content-Type" => "application/json" })
+      end
```

Comment:
I tried moving this stub_request out of the context and below the describe to make the spec thinner but then the stub_request with the 404 which is in the other context would not pass.